roles/mailserver/templates/mailserver.sql.j2

```diff
@@ -21,37 +14,88 @@ CREATE TABLE IF NOT EXISTS "virtual_users" (
         FOREIGN KEY (domain_id) REFERENCES virtual_domains(id) ON DELETE CASCADE
 );
 
-
-CREATE UNIQUE INDEX email_idx ON virtual_users (email);
-
 CREATE TABLE IF NOT EXISTS "virtual_aliases" (
         "id" SERIAL,
         "domain_id" int NOT NULL,
-        "source" TEXT NOT NULL,
+        "source" TEXT NOT NULL UNIQUE,
```

Comment:
This is wrong -- if you want a fan-out alias, you can specify the same source multiple times with different destinations. I make use of this, please don't break it. If you want a unique logical id for this table, you need to use source and destination together.